lib/howl/command.moon

```diff
@@ -158,7 +158,8 @@ class CommandInput
     tab: => @command_completion!
     up: => @run_historical!
     ctrl_r: => @run_historical!
-    escape: => self.finish!
+    binding_for:
+      ["cancel"]: => self.finish!
```

Comment:
Just tried that. Didn't work. I followed the documented instructions on bindings found here http://howl.io/doc/api/bindings.html